Source/WebCore/page/ViewportConfiguration.h

```diff
@@ -148,11 +148,9 @@ class ViewportConfiguration {
     WEBCORE_EXPORT static Parameters imageDocumentParameters();
     WEBCORE_EXPORT static Parameters xhtmlMobileParameters();
     WEBCORE_EXPORT static Parameters testingParameters();
-    
-#if !LOG_DISABLED
+
     String description() const;
     WEBCORE_EXPORT void dump() const;
```

Comment:
Can we keep `dump()` in #if !LOG_DISABLED? It seems not good to introduce `WTFLogAlways()` in release build.
Or maybe drop it if it's unused.

Reply:
> Can we keep dump() in #if !LOG_DISABLED?

Yes, `dump()` is not being used anywhere, so I'll revert that part of the change.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/8d6be03cabc8089fde54db68df858ffd3be1a44e